tests/materials/material_utility_test.cc

```diff
@@ -0,0 +1,213 @@
+#include <cmath>
```

Comment:
You don't need to include all of these again if you are including material_utility.h

Reply:
OK